digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/process/RoadAddressChangeInfoMapperSpec.scala

```diff
@@ -64,6 +65,7 @@ class RoadAddressChangeInfoMapperSpec extends FunSuite with Matchers {
     results.values.flatten.exists(_.startAddrMValue == 1000) should be (true)
     results.values.flatten.exists(_.endAddrMValue == 1000) should be (true)
     results.values.flatten.exists(_.endAddrMValue == 1400) should be (true)
+    results.values.flatten.map(_.commonHistoryId).toSet.size should be (1)
```

Comment:
I think we should check if it still have the same value after the process execute. Not only check if the value is equal. Because let's imagine that we generate a new common history id for all, the test will pass but and it's not suppose.